### PR TITLE
docker installation details on windows

### PR DESCRIPTION
**Pull request by jbmohler**

### CODE REVIEW

File: source/index.html.md [hunk 1]
```diff
@@ -519,57 +519,114 @@ Iterate through the nodes
...
+`C:\` at `/mnt/c/`.   This difference is reconciled by the following configuration
+change.   Create a file `/etc/wsl.conf` (as root via sudo) with the following
+content.
 
-Create a symbolic link to this directory on our linux volume:
+```
+[automount]
+root = /
+options = "metadata"
+```
```
Comment (jonathan-marcus) [inline source/index.html.md]: This ends up on the right-hand pane and not underneath the text as you were hoping. It's hacky but I've had success using two spaces at the end of a line to indicate a line break:
`[automount]  `
`root = /  `
`options = "metadata"  `

File: source/index.html.md [hunk 2]
```diff
@@ -519,57 +519,114 @@ Iterate through the nodes
...
+This requires a restart of WSL so exit your WSL prompt.   Under the Windows
+command prompt or powershell, the command below will restart WSL
+(alternatively, restart the computer).
 
-Set the environment variable CONDUCTO_BASE_DIR to the source on our windows volume (you
-may also want to add this value to your .bashrc file):
+```bash
+# replace <distribution> with the correct value (e.g. Ubuntu-18.04)
+wsl -t <distrbution>
+```
```
Comment (jonathan-marcus) [inline source/index.html.md]: Same as above.

File: source/index.html.md [hunk 3]
```diff
@@ -519,57 +519,114 @@ Iterate through the nodes
...
+it in terms compatible with Docker.
 
-Locate your `.docker/config.json` (typically in your home directory)
+```
+mkdir /c/Users/<user>/.conducto
+ln -s /c/Users/<user>/.conducto ~/.conducto
+ls ~/.conducto
+export CONDUCTO_BASE_DIR=/c/Users/<user>/.conducto >> ~/.bashrc
+source ~/.bashrc
+```
```
Comment (jonathan-marcus) [inline source/index.html.md]: Same as above.

File: source/index.html.md [hunk 4]
```diff
@@ -519,57 +519,114 @@ Iterate through the nodes
...
+in a Windows file system directory shared to WSL.  The following example
+illustrates this.
 
-Delete the `.docker` directory, and create it as a symlink referencing your new
-directory on the windows volume:
+```bash
+mkdir /c/Users/<user>/pipelines
+cd /c/Users/<user>/pipelines
+echo "hello world" > mydata.txt
+```
```
Comment (jonathan-marcus) [inline source/index.html.md]: Same as above. :(

File: source/index.html.md [hunk 5]
```diff
@@ -519,57 +519,114 @@ Iterate through the nodes
...
-may also want to add this value to your .bashrc file):
+```python
+import conducto as do
 
-`export DOCKER_CONFIG_BASE_DIR /mnt/c/Users/jim/Documents/docker`
+def access_local_files() -> do.Serial:
+	output = do.Serial(image=do.Image(context='.'))
+	output['s1'] = do.Exec(f'cat mydata.txt')
+	return output
+```
```
Comment (jonathan-marcus) [inline source/index.html.md]: You can also do `> text` and "text" will appear in the right hand pane above the code block. Maybe that would be better? I don't know.

**Comment by jbmohler** [comment]
indeed, I wasn't happy with that, will try your hints!

**Comment by jbmohler** [comment]
@jonathan-marcus   I did a quick google search (probably the same one you did) and just found people asking the same question about bringing things inline.  However, I found a few ways to re-arrange my docs which I think roll a little better.

More excitingly, my recent push to master means that about half of my docs are now unnecessary.

Let me know what you think.

**Pull request merged**